config/manager/manager.yaml

```diff
@@ -193,3 +193,21 @@ spec:
       scopeName: PriorityClass
       values:
       - system-cluster-critical
+---
```

Comment:
we don't want this file changed since this add to the deployment yaml by default

Reply:
added this in due to the following error. If you have a pointer to resolve that will help, otherwise I can take a look to resolve this later when I get another chance to look. Sorry, not so familiar with the implementation here.
```
Error: no matches for OriginalId autoscaling_v2beta1_HorizontalPodAutoscaler|gatekeeper-system|gatekeeper-controller-manager; no matches for CurrentId autoscaling_v2beta1_HorizontalPodAutoscaler|gatekeeper-system|gatekeeper-controller-manager; failed to find unique target for patch autoscaling_v2beta1_HorizontalPodAutoscaler|gatekeeper-controller-manager
```